jarbas/dashboard/admin.py

```diff
@@ -98,7 +98,11 @@ class HasReceiptFilter(JarbasListFilter):
     )
 
     def queryset(self, request, queryset):
-        return queryset.has_receipt_url(self.value() == 'yes') if self.value() else queryset
+        filter = {
+            'yes': queryset.has_receipt_url(True),
+            'no': queryset.has_receipt_url(False)
+        }
+        return filter.get(self.value(), queryset)
```

Comment:
My bad here. Using `filter` is not recommended (because it overrides the native `filter` function). Choose anything else, even `filter_` instead… so sorry, my mistake not yours ; )

Reply:
Actually I thought about it, but again, forgot to ask. Can I get in contact in the telegram group now for things like this?

Reply:
Sure thing ; )